collector_k8s/templates/collector.yaml

```diff
@@ -1,39 +1,41 @@
+{{ $global := . }}
+{{ range $i, $collector := .Values.collectors -}}
```

Comment:
i believe you can just underscore the `$1` if its unused

Reply:
That doesn't quite work, but a `$_` does the job.